src/screens/Home/components/SignUp/index.tsx

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import React from 'react';
```

Comment:
importing react shouldn't be necessary anymore. Check this in the resto of the components as well